typst.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "cetz"
 version = "0.3.0"
-compiler = "0.11.0"
+compiler = "0.12.0"
```

Comment:
Does this version actually require 0.12.0? The `top-edge` and `bottom-edge` parameters do still exist in 0.11.0

Reply:
Yes, but the real fix for this to work correctly with math content is in Typst 0.12, so I think requiring it is fine.